boltdesignsystem/bolt
# Spacing and Typography | Core | Switch to regular spacing system

## Jira

To be created when vpn works.

## Summary

Reduces spacing and font-size overall.

## Details

1. Redefined the whole spacing scale.
2. Redefined all the font-size maps and variables.
3. Kept the Japanese specific system as its own system and not related to base font-size and spacing.
4. Created SASS variables for the wrapper padding and used them in wrapper, page header, navbar, and toolbar. These variables are using absolute px value so the page max width and padding are immune to font-size and spacing changes.

## How to test

Run the branch locally and QA all pages, make sure things still line up and there is no awkward white space.

## Notes

Drupal needs to update the mega nav's padding. I should be doing this, will need @remydenton's help.
